### PR TITLE
Restaurant page

### LINKED ISSUE #2: Restaurant -  Initial Page
Description:
- Create an html page 
- General informaton about restaurant (ex. opening hours)
- Palceholder for a picture of restaurant part

User story:
- As a user I want to see how restaurant looks like and when it is open.

Dependencies:
- #5 

Acceptance criteria:
- After press on restaurant button in navigation menu changes content on the page into information about restaurnat and opening hours
- Site looks good on desktop and mobile

Design:

https://www.figma.com/file/vhNtZfxUiEfTYAYr6yaExM/VadSomHelst?node-id=0%3A1&t=JeYxmP7CNyyORHG8-0

### PR DESCRIPTION
**Pull request by matodorovic**
### Description
This PR adds  restaurant page.

Screenshots
Skärmavbild 2023-01-02 kl  19 55 09
Skärmavbild 2023-01-02 kl  19 56 38

### Changes
- Adds restaurant page with some styling for desktop and mobile
- Adds links in all html pages to restaurant page

### Issues:
#2

### CODE REVIEW

File: about-us.html [hunk 1]
```diff
@@ -63,7 +63,7 @@
         <li class="movies-nav">
           <ul class="movies-nav-list">
             <li class="movies-nav-element">
-              <a href="#">Restaurang <span class="arrow-top">&#9662;</span></a>
+              <a href="/restaurant.html">Restaurang <span class="arrow-top">&#9662;</span></a>
```
Comment (mkor92) [inline about-us.html:66]: We need to remove " / " at the start of all links. Github pages doesnt link otherwise.

**Review by mkor92 — APPROVED** [review]
Nicely done Marta! I added a comment on a thing we all need to look over before pushing.

File: about-us.html (hunk 1 above)
Comment (matodorovic) [reply about-us.html:66]: I missed those in this PR. I fixed almost all / on other PR. I will fix that tomorrow.

**Review by jakobpalmborg — APPROVED** [review]
Great work!

**Pull request merged**